content/courses/program-optimization/program-configuration.md

```diff
@@ -955,67 +947,72 @@ it("Payment completes successfully", async () => {
 });
 ```
 
-The third test attempts to update the fee on the program config account, which
+The third test attempts to update the fee on the admin config account, which
 should be successful.
 
 ```typescript
-it("Update Program Config Account", async () => {
+it("Admin Config Update should be successfully", async () => {
   const tx = await program.methods
-    .updateProgramConfig(new anchor.BN(200))
+    .updateAdminConfig(new anchor.BN(200))
     .accounts({
-      programConfig: programConfig,
-      admin: wallet.publicKey,
       feeDestination: feeDestination,
       newAdmin: sender.publicKey,
     })
     .rpc();
 
   assert.strictEqual(
     (
-      await program.account.programConfig.fetch(programConfig)
+      await program.account.adminConfig.fetch(adminConfig)
     ).feeBasisPoints.toNumber(),
     200,
   );
 });
 ```
 
-The fourth test tries to update the fee on the program config account, where the
-admin is not the one stored on the program config account, and this should fail.
+The fourth test tries to update the fee on the admin config account, where the
+admin is not the one stored on the admin config account, and this should fail.
 
 ```typescript
-it("Update Program Config Account with unauthorized admin (expect fail)", async () => {
+it("Admin Config Update with unauthorized admin should throw an exception", async () => {
   try {
     const tx = await program.methods
```

Comment:
Use transaction, instruction, signature,  error, etc everywhere. See 'use full names' in https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#code